src/PageContent/Calendar/DialogContent.js

```diff
@@ -8,25 +9,41 @@ import DonatingAgencyContent from './DonatingAgencyContent';
 class DialogContent extends Component {
     render() {
         return (
-            <div>
-                <DescriptionContent
-                    accountType={this.props.accountType}
+            <div className="dialog-content-wrapper">
+                <DelivererGroupContent
                     futureEvent={this.props.futureEvent}
                     delivery={this.props.delivery}
-                />
-                <ContactContent
+                    accountOwnerName={this.props.accountOwnerName}
                     accountType={this.props.accountType}
-                    futureEvent={this.props.futureEvent}
-                    delivery={this.props.delivery}
                 />
-                <DelivererGroupContent
+                {this.props.accountType === AccountType.DELIVERER_GROUP ? (
```

Comment:
you didn't need to add this new case. The original conditional rendering was better - no redundancy. I think all you had to do was to move the `if not DA, show DA content` block to be above the `if not RA, show RA content` block.